PREPARATION.md

```diff
@@ -0,0 +1,96 @@
+
+* the `op-scim` service itself
+* a [Redis](https://redis.io/) cache
+* a load balancer or web server to handle TLS connections on port 443
+
+
+### SSL Certificates
+
+SSL certificates are handled through the [https://letsencrypt.org/](LetsEncrypt) service which automatically generates and renews an SSL certificate based on the domain name you've decided on. On your firewall, you should ensure that the service can access Port 80 and Port 443, as Port 80 is required for the LetsEncrypt service to complete its domain challenge and issue your SCIM bridge an SSL certificate. Note that a TLS connection is still mandatory for connecting to the 1Password service.
+
+
+## Clone this repository
+
+You should clone this repository to ensure you have all the files needed to begin deployment. You should also familiarize yourself with the contents of the deployment method you've selected to ensure you have a full idea of what the deployment process will do.
+
+From the command line:
+
+```
+git clone https://github.com/1Password/scim-examples.git
+```
+
+Alternatively, you can download a .zip of the project by clicking the "Clone or download" button.
+
+
+## Caveats
+
+There are a few common issues that pop up when deploying the SCIM Bridge.
+
+* Do not create the Provision Manager user manually. Let the setup process create the Provision Manager user for you **automatically.**
+* When the Provisioning setup asks you for an email address for the new Provision Manager user it creates for you automatically, use a **dedicated email address** (for example: `op-provision-manager@example.com`) to handle this account. It is _not advised_ to use any personal email address.
+* You should **never** need to log into this Provision Manager account manually.
+* Do not attempt to perform a provisioning sync until the setup has been completed.
+* Once set up, your Identity Provider becomes the _authoritative source_ of information for your 1Password accounts. With Provisioning enabled, you _cannot edit user details manually_, and must do so through your Identity Provider.
+
+
+## Prepare your 1Password Account
+
+Log in to your 1Password account [using this link](https://start.1password.com/settings/provisioning/setup). It will take you to the setup page for the SCIM bridge. Follow the instructions there.
+
+During this process, the setup will guide you through the following process:
+
+* Automatically creating a Provision Managers group
+* Automatically creating a Provision Manager user
+* Generating your SCIM bridge credentials
+
+
+### Security (IMPORTANT)
+
+There are a few specific considerations with respect to security.
+
+All SCIM requests must be secured via TLS using an API gateway (self-configured web server) or the provided load balancer.
+
+Anonymous access to 1Password is not supported. You must use the provided secrets to authenticate with the SCIM Bridge and 1Password service.
```

Comment:
Can you clarify to me what you are trying to communicate here?

Reply:
Removed entirely. Wasn't needed.